scripts/rest_examples/reservation-create-aws.http

```diff
@@ -1,9 +1,14 @@
 // @no-log
-POST http://{{hostname}}:{{port}}/{{prefix}}/reservations/aws HTTP/1.1
+POST http://localhost:8005/api/provisioning/v1/reservations/aws HTTP/1.1
 Content-Type: application/json
 X-Rh-Identity: {{identity}}
 
 {
+
+  "source_id": 9, 
+  "ami": "ami-0cff7528ff583bf9a",
```

Comment:
I think this should not be ami, but image_id...
can we just drop change to this file for now and add it only once we are addressing the endpoint itself and have #116 in? :)

Reply:
So should I change this to image_id? just the for the reservation  model

Reply:
I'd just drop the change of this file... we need to change the endpoint first anyway :)

Reply:
second thought, lets just improve on this file when we are addressing the endpoint itself :+1: